src/encoded/types/biosample.py

```diff
@@ -513,6 +513,30 @@ def summary(self, request,
         return construct_biosample_summary([biosample_dictionary],
                                            sentence_parts)
 
+    @calculated_property(schema={
+        "title": "Perturbed",
+        "description": "A flag to indicate whether the biosample has been perturbed with a treatment or genetic modification.",
+        "type": "boolean",
+        "notSubmittable": True,
+    })
+    def perturbed(
+        self,
+        request,
+        treatments=None,
+        genetic_modifications=None,
+        model_organism_donor_modifications=None
+    ):
+        applied_modifications = get_applied_modifications(
+            genetic_modifications, model_organism_donor_modifications)
```

Comment:
Is it worth having this property condition on applied_modifications so that `get_applied_modifications` doesn't have to be called twice, @keenangraham ?

Reply:
It looks like the calculated applied_modifications property already does this exact calculation, so you should be able to add `define=True` to applied_modifications https://github.com/ENCODE-DCC/encoded/blob/3b97e8bf8e15511899737f2c70471523ca7e4fa7/src/encoded/types/biosample.py#L270 and use it in your function signature: 
```
def perturbed(
        self,
        request,
        applied_modifications,
        treatments=None,
    ):
```

Reply:
Added define=True and removed the get_applied_modifications section